### ISSUE TITLE
Add List of LLM's Supported

### ISSUE THREAD

**Issue by haseeb-heaven** [zozoheir/tinyllm#20]
Please add list of all **LLM's** supported and all the one with **future** updates.
There are lots of details missing with **Docs** as well.
Please add them.

Thanks

**Comment by zozoheir** [comment]
tinyllm uses litellm to make LLM calls https://github.com/BerriAI/litellm/
litellm aggregates more than 20+ model providers. in tinyllm you can just set the model argument to whatever model you need like 
```
agent(user_input='hii', model='gpt-4') or  agent(user_input='hii', model='hugginface/model_name')
```

**Comment by haseeb-heaven** [comment]
Great then please add more documentation on README then about these examples with models.

**Comment by zozoheir** [comment]
Yes, I will add proper documentation when the library is fully stable.
